yace.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: yace
   version: 0.62.1
-  epoch: 7 # CVE-2025-47907
+  epoch: 8 # CVE-2025-47907
```

Comment:
Small nit, but I prefer to remove the `# CVE...` comment when I bump epochs, otherwise it might give the impression that this bump is fixing the CVE.